api/admin_test.go

```diff
@@ -235,6 +235,10 @@ func TestGetPostCount(t *testing.T) {
 	post1 := &model.Post{ChannelId: channel1.Id, Message: "a" + model.NewId() + "a"}
 	post1 = Client.Must(Client.CreatePost(post1)).Data.(*model.Post)
 
+	// manually update creation time, since it's always set to 0 upon saving and we only retrieve posts < today
```

Comment:
hah...the comment threw me.  Should be something like, manually update creation time to yesterday.